src/solve/Solver.h

```diff
@@ -99,6 +121,11 @@ class Solver {
   std::vector<double> times;
   State initial_state;
 
+  // Time integration variables added as class variables
```

Comment:
Remove comment and group new variables with the ones above (e.g. put the states together)